src/Features.php

```diff
@@ -40,6 +40,8 @@ class Features {
 
 	public const ALLOW_ALL_BLOCKS = 'allow_all_blocks';
 
+	public const ACTION_POST_TYPE = 'action_post_type';
+
```

Comment:
This should be move in the new "Information Architecture" menu.

See: https://github.com/greenpeace/planet4-master-theme/blob/f33182d785345b8ff7aecdfd290b1ff991ae58ef/src/Settings/InformationArchitecture.php

Reply:
Done 👍